lib/simp/beaker_helpers.rb

```diff
@@ -165,7 +165,7 @@ def copy_fixture_modules_to( suts = hosts, opts = {})
     opts[:pluginsync] = opts.fetch(:pluginsync, true)
 
     unless ENV['BEAKER_copy_fixtures'] == 'no'
-      block_on(suts, :run_in_parallel => true) do |sut|
+      block_on(suts, :run_in_parallel => false) do |sut|
```

Comment:
Why not set this to an environment variable test that defaults to `false`?

You could use a new one, like `BEAKER_SIMP_parallel=yes` or (if you're feeling fancy) a new list item for [`BEAKER_RUN_IN_PARALLEL`](https://github.com/puppetlabs/beaker/blob/master/docs/how_to/run_in_parallel.md#run_in_parallel-global-option), like "`simp_helpers`".

Reply:
I'll add the environment variable